app/src/main/java/com/cornellappdev/score/components/SportCard.kt

```diff
@@ -0,0 +1,137 @@
+            ) {
+                // Team Logo and Team Name
+                Row(
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Image(
+                        painter = teamLogo,
+                        contentDescription = "Team Logo",
+                        modifier = Modifier
+                            .height(20.dp)
+                            .padding(start = 4.dp, end = 4.dp)
+                    )
+                    Text(
+                        text = team,
+                        style = teamName
+                    )
+                }
+
+                // Date Text
+                Text(
+                    text = date,
+                    style = dateText
+                )
+            }
+
+            // Second Row: Location, Sport Name, Sport Icon
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 12.dp) // Add top padding to create space
+            ) {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_location), // Location icon
+                        contentDescription = "Location Icon",
+                        modifier = Modifier
+                            .padding(1.dp)
+                            .width(24.dp)
+                            .height(24.dp)
+                    )
+                    Text(
+                        text = location,
+                        style = universityText,
+                        modifier = Modifier.padding(start = 4.dp)
+                    )
+                }
+                Row(
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    Text(
+                        text = sportName,
+                        style = Style.sportName,
+                        modifier= Modifier.padding(end = 4.dp)
+                    )
+                    Image(
+                        painter = sportIcon,
+                        contentDescription = "Sport Icon",
+                        modifier = Modifier
+                            .padding(1.dp)
+                            .width(24.dp)
+                            .height(24.dp)
+                    )
+                }
+            }
+        }
+    }
+}
+
+
+@Preview
+@Composable
+fun SportCardPreview() {
+    Column {
+        SportCard(
+            teamLogo = painterResource(id = R.drawable.penn_logo),
+            team = "Penn",
+            date = "5/20/2024",
+            location = "U. Pennsylvania",
+            sportName = "Baseball",
+            sportIcon = painterResource(id = R.drawable.ic_baseball)
+        )
+    }
```

Comment:
Love it! Argument choice is really nice, and the preview looks great. I like that you're taking in the painters as parameters.

Reply:
Addnmodifier as parameter and padding.